tools/excel/nist/sp-800-53/nist-sp-800-53-rev5_to_iso27001-2022.py

```
@@ -0,0 +1,65 @@
+        # Extract column A (index 0) and column D (index 3)
+        temp_df = df.iloc[:, [0, 3]]
+        temp_df.columns = ['source_node_id', 'target_node_id']
+
+        # Drop rows where target_node_id is missing
+        temp_df = temp_df.dropna(subset=['target_node_id'])
+
+        # Function to clean values from column A (source_node_id)
+        def clean_source(value):
+            if isinstance(value, str):
+                value = value.strip().lower()                     # Lowercase and remove spaces
+                value = re.sub(r'-(0)(\d)\b', r'-\2', value)      # Remove unnecessary leading zeros (e.g., PX-01 → PX-1)
```

Pylint (3.3.7)

[convention] Line too long (121/100)

(C0301)